frontend/package.json

```diff
@@ -6,9 +6,15 @@
     "@testing-library/jest-dom": "^5.11.5",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "bootstrap": "^4.5.3",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.5",
+    "jsdom": "^16.4.0",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-scripts": "4.0.0",
+    "react-test-renderer": "^17.0.1",
+    "reactstrap": "^8.7.0",
```

Comment:
I think this is the reason why you had a merge conflict. Did you download another package? I think you used this command: `npm install --save enzyme enzyme-adapter-react-16 react-test-renderer jsdom`. Maybe you can revert the changes before you used the command and it will fix the merge error

Reply:
yeah, I don't think I ran npm update or something like that before pushing the branch. It's fixed now I think